plugins/vendor/zoom/lib/model/ZoomVendorIntegration.php

```diff
@@ -179,24 +179,26 @@ public function getGroupParticipationType()
 		return $this->getFromCustomData(self::GROUP_PARTICIPATION_TYPE, null, kZoomGroupParticipationType::NO_CLASSIFICATION);
 	}
 	
-	public function shouldExcludeUserRecordingsIngest($puserId)
+	public function shouldExcludeUserRecordingsIngest($userId)
```

Comment:
your function gets a non consistent value (sometimes string puser id and sometimes inte kuser id).
so your previous logic won't work and the current one won't work for all cases as well.
you need to return back the previous logic and change https://github.com/kaltura/server/blob/e6d76c6cadeaefb59955ba83160fe76035fe700c/plugins/vendor/zoom/lib/model/kZoomEventHanlder.php#L142 to get puser id like we do on zoom processor class